tests/platform_tests/daemon/test_chassisd.py

```diff
@@ -0,0 +1,211 @@
+    check_critical_processes(duthost, watch_secs=10)
+
+
+@pytest.fixture
+def check_daemon_status(duthosts, enum_rand_one_per_hwsku_hostname):
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    daemon_status, daemon_pid = duthost.get_pmon_daemon_status(daemon_name)
+    if daemon_status is not "RUNNING":
+        duthost.start_pmon_daemon(daemon_name)
+        time.sleep(10)
+
+def check_expected_daemon_status(duthost, expected_daemon_status):
+    daemon_status, _ = duthost.get_pmon_daemon_status(daemon_name)
+    return daemon_status == expected_daemon_status
+
+def collect_data(duthost):
+    keys = duthost.shell('sonic-db-cli STATE_DB KEYS "CHASSIS_*TABLE|*"')['stdout_lines']
+
+    dev_data = {}
+    for k in keys:
+        data = duthost.shell('sonic-db-cli STATE_DB HGETALL "{}"'.format(k))['stdout_lines']
+        data = compose_dict_from_cli(data)
+        dev_data[k] = data
+    return {'keys': keys, 'data': dev_data}
+
+def wait_data(duthost):
+    class shared_scope:
+        data_after_restart = {}
+    def _collect_data():
+        shared_scope.data_after_restart = collect_data(duthost)
+        return bool(shared_scope.data_after_restart['data'])
+    pooling_interval = 60
+    wait_until(pooling_interval, 10, 20, _collect_data)
+    return shared_scope.data_after_restart
+
+@pytest.fixture(scope='module')
+def data_before_restart(duthosts, enum_rand_one_per_hwsku_hostname):
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+
+    data = collect_data(duthost)
+    return data
+
+
+def test_pmon_chassisd_running_status(duthosts, enum_rand_one_per_hwsku_hostname, data_before_restart):
+    """
+    @summary: This test case is to check chassisd status on dut
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+
+    skip_release(duthost, ["201811", "201911", "202012"])
+
+    daemon_status, daemon_pid = duthost.get_pmon_daemon_status(daemon_name)
+    logger.info("{} daemon is {} with pid {}".format(daemon_name, daemon_status, daemon_pid))
+    pytest_assert(daemon_status == expected_running_status,
+                          "{} expected running status is {} but is {}".format(daemon_name, expected_running_status, daemon_status))
+    pytest_assert(daemon_pid != -1,
+                          "{} expected pid is a positive integer but is {}".format(daemon_name, daemon_pid))
+
+    pytest_assert(data_before_restart['keys'], "DB keys is not availale on daemon running")
+    pytest_assert(data_before_restart['data'], "DB data is not availale on daemon running")
+
+
+def test_pmon_chassisd_stop_and_start_status(check_daemon_status, duthosts, enum_rand_one_per_hwsku_hostname, data_before_restart):
+    """
+    @summary: This test case is to check the chassisd stopped and restarted status
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+
+    skip_release(duthost, ["201811", "201911", "202012"])
```

Comment:
Can the skip be added to the conditional mark file? Then the skipping can be processed in an earlier stage without having to run some fixtures unnecessarily.
Location of the conditional marks definition file: https://github.com/Azure/sonic-mgmt/tree/master/tests/common/plugins/conditional_mark

Reply:
Updated, please check